packages/shortest/src/types/config.ts

```diff
@@ -41,10 +42,18 @@ const mailosaurSchema = z
 
 const testPatternSchema = z.string().default("**/*.test.ts");
 
+const browserSchema = z.object({
+  /**
+   * @see https://playwright.dev/docs/api/class-browser#browser-new-context
+   */
+  contextOptions: z.custom<BrowserContextOptions>().optional(),
+});
+
 export const configSchema = z
   .object({
     headless: z.boolean().default(true),
     baseUrl: z.string().url("must be a valid URL"),
```

Comment:
This should be moved into `browserSchema`, and ideally made optional (with default `about:blank`).